### PR TITLE
Create flip

### PR DESCRIPTION
**Pull request by jfioasd**

### CODE REVIEW

File: languages.json [hunk 1]
```diff
@@ -237,6 +237,13 @@
         "url": "https://github.com/PyGamer0/flax",
         "sbcs": true
     },
+    "Flip": {
```
Comment (pxeger) [inline languages.json]: The ID here needs to match the runner filename, and it's case-sensitive.

```suggestion
    "flip": {
```

File: runners/flip [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```
Comment (pxeger) [inline runners/flip:1]: This file needs to be marked as executable (`chmod +x runners/flip`)

File: runners/flip [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+
+cd /ATO/context
+python3 /opt/flip/flip.py /ATO/code < /ATO/input
```
Comment (pxeger) [inline runners/flip]: Please follow the standard format for these runners, even if the language does not accept any arguments or options (yet).

```suggestion
/ATO/yargs %1 /ATO/options /ATO/yargs %2 /ATO/arguments python /opt/flip/flip.py %1 /ATO/code %2 < /ATO/input
```

**Review by pxeger — APPROVED** [review]
You did a couple of things slightly wrong here - please make sure to read the [contributing docs](https://github.com/attempt-this-online/attempt-this-online/blob/main/docs/contributing.md). I've fixed them for you this time though

File: languages.json (hunk 1 above)
Comment (jfioasd) [reply languages.json]: Yeah, I never knew what the identifier in languages.json was actually for. Maybe you should clarify the `identifier-safe` thing in the contributing docs?

File: runners/flip (hunk 2 above)
Comment (jfioasd) [reply runners/flip:1]: Well, GitHub doesn't let you upload executable files (whenever I try to use git to commit, all the executable information is lost, so marking `chmod +x` doesn't really help). Can you explain how you did it?

Comment (jfioasd) [reply runners/flip:1]: Finally knew how to do it. https://www.augmentedmind.de/2020/05/10/commit-executable-shell-scripts-with-git/

File: languages.json (hunk 1 above)
Comment (pxeger) [reply languages.json]: Ah, sorry, you're right, that's never specified anywhere.

**Pull request merged**